contracts/self-updates/update/README.md

```diff
@@ -5,43 +5,61 @@ in the `PostedMessage` structure.
 
 ```rust
 pub struct PostedMessage {
-  pub payment: u128, 
-  pub premium: bool, 
-  pub sender: AccountId,
-  pub text: String
+    pub payment: NearToken,
+    pub premium: bool,
+    pub sender: AccountId,
+    pub text: String,
 }
 
 pub struct GuestBook {
-  messages: Vector<PostedMessage>,
+    messages: Vector<PostedMessage>,
+    manager: AccountId,
 }
 ```
 
-## 1. Asking the Contract to Update Itself
+## 1. Build
+
+Install [`cargo-near`](https://github.com/near/cargo-near) and run:
+
+```bash
+# from repo root
+cd contracts/self-updates/update
+cargo near build
+```
+
+## 2. Asking the Contract to Update Itself
+
+_In this example we will be using [NEAR CLI](https://github.com/near/near-cli)
+to intract with the NEAR blockchain and the smart contract and [near-cli-rs](https://near.cli.rs)
+which provides more control over interactions and has interactive menus for subcommands selection_
 
 The [base contract](../base/) implements a `update_contract` method that only the `manager` can call. That method takes
 a compiled wasm as input and then:
-1. Deploys in on itself.
+1. Deploys it on itself.
 2. Calls the `migrate` method on itself.
 
 Lets call `update_contract` passing the new code ([./src](./src/)) using the [`manager-account`](../base/README.md#1-build-and-deploy-the-contract).
 
 ```bash
-# run from project-root/contracts
-NEW_CONTRACT_BYTES=`cat ./target/wasm32-unknown-unknown/release/self_update.wasm | base64`
-near call <dev-account> update_contract "$NEW_CONTRACT_BYTES" --base64 --accountId <manager-account> --gas 300000000000000
```

Comment:
`NEAR CLI` example command was removed for this case, 
as it was a similar error both for `zsh` and `bash`:
```bash
zsh: argument list too long: near
```

probably that may be caused by arg string being too long (https://unix.stackexchange.com/questions/606847/argument-list-too-long-in-zsh-with-any-command):
```bash
❯ echo $NEW_CONTRACT_BYTES | wc -c
151877
```